CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
 ## 1.3.3 (TBD)
+* Breaking changes
```

Comment:
In the future whenever we release a big new feature like commands sets we should advertise that feature as being in some kind of alpha/beta/release-candidate phase where it is subject to change and keep it that way for a few minor releases until we feel it is stable.  We should come up with a formalized way of doing this.